ports/stm32/boards/openocd_stm32f4.cfg

```diff
@@ -11,7 +11,7 @@
 #    $ openocd -f openocd_stm32f4.cfg
 
 
-source [find interface/stlink-v2.cfg]
+source [find interface/stlink-v2-1.cfg]
```

Comment:
I think this will break older boards, like the STM32F4DISC board.  I'll need to check it.  Probably the way to make it work for all boards is to have 2 of these config files, one for v2 and one for v2-1.

Reply:
This setting should therefore probably be tied to the board and not to the processor.

Reply:
I tested this PR with an STM32F4DISC board and this new config doesn't work with this older board.  So there will need to be multiple of these open OCD config files.